.rubocop.yml

```diff
@@ -1,10 +1,10 @@
 AllCops:
+  TargetRubyVersion: 1.9
```

Comment:
Would love to, but can't immediately as a lot of our projects are still using older versions of JRuby/MRI - once we've updated, then yes we will 👍